packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/project/services/DefaultEditingContextActionProvider.java

```diff
@@ -29,8 +34,21 @@ public class DefaultEditingContextActionProvider implements IEditingContextActio
 
     public static final String EMPTY_ACTION_ID = "empty";
 
+    private final IProjectSearchService projectSearchService;
+
+    public DefaultEditingContextActionProvider(IProjectSearchService projectSearchService) {
+        this.projectSearchService = Objects.requireNonNull(projectSearchService);
+    }
+
     @Override
     public List<EditingContextAction> getEditingContextAction(IEditingContext editingContext) {
-        return List.of(new EditingContextAction(EMPTY_ACTION_ID, "Others..."));
+        var isWithoutNature = this.projectSearchService.findById(UUID.fromString(editingContext.getId()))
```

Comment:
new UUIDParser().parse(...), it ensures that the exception is taken into account

Reply:
Done